fluss-server/src/main/java/com/alibaba/fluss/server/metrics/group/PhysicalTableMetricGroup.java

```diff
@@ -78,8 +78,12 @@ public PhysicalTableMetricGroup(
     protected void putVariables(Map<String, String> variables) {
         variables.put("database", physicalTablePath.getDatabaseName());
         variables.put("table", physicalTablePath.getTableName());
+
         if (physicalTablePath.getPartitionName() != null) {
             variables.put("partition", physicalTablePath.getPartitionName());
+        } else {
+            LOG.debug("Setting variable 'partition' for non-partitioned table to empty string.");
```

Comment:
nit: I don't think we should log it since it doesn't provide any useful info... We all know the partition will be empty string for non-partitioned table... Maybe you can add a comment about it in code instead of log it.